src/psyclone/psyGen.py

```diff
@@ -42,8 +42,20 @@
     particular API and implementation. '''
 
 import abc
-import psyclone
-import config
+from psyclone import config
+
+# We use the termcolor module (if available) to enable us to produce
+# coloured, textual representations of Invoke schedules. If it's not
+# available then we don't use colour.
+try:
+    from termcolor import colored
+except ImportError:
+    # We don't have the termcolor package available so provide
+    # alternative routine
+    def colored(text, color):
+        ''' Returns the supplied text argument unchanged. Is a swap-in
```

Comment:
Please change "Is" to "This is"

Reply:
Done, plus, inspired by the RSE conference, I've extended the docstring to fully document the interface.